src/server/views/help.pug

```diff
@@ -0,0 +1,54 @@
+extends includes/layout.pug
+block title
+    title Help - NetsBlox
+block content
+  mixin box(title, url)
+    .col-md-4.col-sm-6.text-center
+      .box
+        h2.tag-title= title
+        block          
+        br
+        a.btn.btn-block.btn-primary(href=url) Learn more
+  .container
+    .row
+      h2.text-center Learn more about NetsBlox
+      .row
+        +box("How To's",'/howtos')
+          p Guides and tutorials to help you understand and use NetsBlox.
+          .text-left
+            h4 How to:
+            ul
+              li 
+                a(href='/howtos/#use-services') Use Services book (Remote Procedure Calls)
+              li
+                a(href='/howtos/#manage-room') Manage rooms
+              li
+                a(href='/howtos/#send-messages') Send and receive messages
+            h4 What are: 
+            ul
+              li
+                a(href='/howtos/#rooms') Rooms
+              li
+                a(href='/howtos/#messages') Messages
+              li 
+                a(href='/howtos/#services') Services (RPCs)
+        +box('QuickStart','/quickstart')
+          p Get started with playing multiplayer games on NetsBlox in no time.
+          
+        +box('Demos','/tutorials')
+          p Sit back and watch walkthroughs and demos of NetsBlox here.
```

Comment:
A little odd phrasing in this context. I would say something like "Check out walkthroughs and featured demos of Netsblox here"